src/screens/ModelPage/ModelPage.js

```diff
@@ -62,37 +64,34 @@ function ModelPage() {
   }, [countyOption]);
   const history = useHistory();
 
-  const projections = useProjections(
-    _location,
-    selectedCounty
-  );
-  const stateSummary = useStateSummary(_location);
+  const projections = useProjections(stateId, selectedCounty);
+  const stateSummary = useStateSummary(stateId);
 
-  const locationName = STATES[_location];
-  const intervention = STATE_TO_INTERVENTION[_location];
+  const stateName = STATES[stateId];
+  const intervention = STATE_TO_INTERVENTION[stateId];
 
   const goTo = route => {
     history.push(route);
   };
 
   // No projections
-  if (!projections || projections.county != selectedCounty) {
+  if (!projections || projections.county !== selectedCounty) {
     return <LoadingScreen></LoadingScreen>;
   }
 
   let actionTitle;
   let actionDescription;
   if (intervention === INTERVENTIONS.SHELTER_IN_PLACE) {
-    actionTitle = `${locationName}: Keep staying at home to protect against the COVID-19 outbreak.`;
+    actionTitle = `${stateName}: Keep staying at home to protect against the COVID-19 outbreak.`;
     actionDescription = `Avoiding hospital overload depends heavily on your cooperation.`;
   } else {
-    actionTitle = `${locationName}: Urge your public officials to act now against the COVID-19 outbreak!`;
+    actionTitle = `${stateName}: Urge your public officials to act now against the COVID-19 outbreak!`;
     actionDescription = `To prevent hospital overload, our projections indicate a Stay at home order must be implemented soon.`;
   }
   let metaTags = (
     <AppMetaTags
-      canonicalUrl={`/us/${_location.toLowerCase()}`}
-      pageTitle={`${locationName} Forecast`}
+      canonicalUrl={`/us/${stateId.toLowerCase()}`}
+      pageTitle={`${stateName} Forecast`}
```

Comment:
Huh.  I guess none of these meta tags respect the county at all.  Logged a bug. https://github.com/covid-projections/covid-projections/issues/568